portal/server/controllers/utilisation-report-service/utilisation-report-upload/utilisation-report-details.js

```diff
@@ -0,0 +1,43 @@
+const { format, parseISO } = require('date-fns');
+
+/**
+ * @typedef {Object} ReportAndUserDetails
+ * @property {string} uploadedByFullName - The index of the object with format '{firstname} {surname}'
+ * @property {string} formattedDateAndTime - The date uploaded formatted as 'h:mmaaa'
+ * @property {string} lastUploadedReportPeriod - The report period of the report formatted as 'MMMM yyyy'
```

Comment:
Part of the description for `uploadedByFullName` seems to be copied from elsewhere (re 'index of the object').

The description for `formattedDateAndTime` should include the date format too.